src/atoms/copy/index.tsx

```diff
@@ -20,8 +20,9 @@ interface CopyProxyProps {
 }
 
 export enum Size {
+	Default,
 	Small,
-	Medium
```

Comment:
I would consider leaving Medium as a default value, instead of including a `Default` size.
You can still use `Medium` in the switch statement and combine it with default.

The reason for my suggestion has a lot to do with the consumption of the **Design Kit** as a user. I think it's confusing for the user not to have a logical enumerate properties

Reply:
you are completely right